docs/narr/MyProject/development.ini

```diff
@@ -1,6 +1,6 @@
 ###
 # app configuration
-# http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html
+# http://docs.pylonsproject.org/projects/pyramid/en/1.7-branch/narr/environment.html
```

Comment:
Ah, I wondered why when I clicked the link in the email notification, I couldn't @mmerickel's comment.